Widget/Fitness/FitnessWidget.swift

```diff
@@ -0,0 +1,238 @@
+                    
+                }
+            }
+            .frame(width: 90, height: 90)
+            
+            Text(lastUpdatedFormattedTime(date: rooms[0].last_updated))
+                .font(.system(size: 9))
+                .foregroundStyle(.gray)
+            
+            Spacer()
+        }.widgetPadding()
+            .frame(maxWidth: .infinity, maxHeight: .infinity)
+    }
+    
+}
+
+private struct FitnessWidgetMediumView: View {
+    var rooms: [FitnessRoom]
+    var configuration: ConfigureFitnessWidgetIntent.Configuration
+    
+    var body: some View {
+        VStack {
+            HStack {
+                Text(rooms[0].name)
+                    .font(.system(size: 12, weight: .semibold))
+                    .padding(.top)
+                
+                Text("")
+                
+                Text(getOpenString(room: rooms[0], showbusystring: true))
+                    .font(.system(size: 12))
+                    .foregroundStyle(isOpen(room: rooms[0]) ? .green : .blue)
+                    .multilineTextAlignment(.center)
+                    .fixedSize(horizontal: false, vertical: true)
+                    .padding(.top)
+            }
+            .padding([.trailing, .leading], 10)
+            HStack {
+                VStack {
+                    MeterView(current: isOpen(room: rooms[0]) ? rooms[0].capacity : 0, maximum: 100.0, style: Color.blue, lineWidth: 6) {
+                        VStack {
+                            Text("\(isOpen(room: rooms[0]) ? rooms[0].capacity : 0, specifier: "%.2f")%")
+                                .fontWeight(.bold)
+                            Text("capacity")
+                                .font(.system(size: 10, weight: .light, design: .default))
+                            
+                        }
+                    }
+                    .frame(width: 90, height: 90)
+                    
+                    Text(lastUpdatedFormattedTime(date: rooms[0].last_updated))
+                        .font(.system(size: 9))
+                        .foregroundStyle(.gray)
+                    
+
+                    
+                    Spacer()
+                }
+                .padding(.trailing, 7)
+                
+                FitnessGraph(room: rooms[0])
+                
+                Spacer()
+            }
+            .padding([.trailing, .leading], 10)
+        }.widgetPadding()
+            .frame(maxWidth: .infinity, maxHeight: .infinity)
+    }
+}
+
+private struct FitnessHomeWidgetView: View {
+    var entry: FitnessEntry<ConfigureFitnessWidgetIntent.Configuration>
+    
+    @Environment(\.widgetFamily) var widgetFamily
+    
+    var body: some View {
+        Group {
+            if (entry.configuration.complex.rawValue == 0) {
+                (Text("To use this widget, choose a room by ") +
+                Text("touching and holding the widget")
```

Comment:
Can we keep these instructions the first time you add the widget (how it is right now), but change the preview display to be a specific room example (when selecting the widget)?

Reply:
If this is too much of a blocker, I think the current behavior is fine (after all, it's what apps like Widgetsmith do for unconfigured widgets.)

Reply:
I was able to do this!